app/assets/stylesheets/pages/_playbook.scss

```diff
@@ -0,0 +1,80 @@
+.pb-bg-navy { background-color: #163963;}
```

Comment:
why are we adding another shade of navy? can we standardize on one and then simply use the `bg-navy` class?